examples/cli/src/main/java/io/unitycatalog/cli/CatalogCli.java

```diff
@@ -69,6 +70,9 @@ private static String getCatalog(CatalogsApi catalogsApi, JSONObject json) throw
     private static String updateCatalog(CatalogsApi apiClient, JSONObject json) throws JsonProcessingException, ApiException {
         String catalogName = json.getString(NAME_PARAM);
         json.remove(NAME_PARAM);
+        if (json.length() == 0) {
+            throw new CliException("Nothing to update.");
```

Comment:
Agree that this needs to be more descriptive. Something along the lines: No parameters to update, please provide one of: [[new_name, ..]](https://github.com/unitycatalog/unitycatalog/blob/main/examples/cli/src/main/java/io/unitycatalog/cli/utils/CliUtils.java#L79) to update.